### ISSUE TITLE
Conference does not use german voice prompts

### ISSUE THREAD

**Issue by jpawlowski** [amooma/GS5#168]
it seems the conference currently does not respect default_language setting.
Even though I set default_language=de the conference system is still in english, voicemail system is german as expected.

**Comment by spag** [comment]
fixed in commit 2fc71d2c420fcb328165f91806080367a9fa2928

**Comment by jpawlowski** [comment]
There seems to be something missing :-)

```
2013-02-18 22:47:34.175886 [DEBUG] freeswitch_lua.cpp:352 DBH handle 0x8951bc8 Connected.
2013-02-18 22:47:34.175886 [DEBUG] switch_cpp.cpp:1274 #C# [conference.conf] CONFIG_CONFERENCE 1 name: Telefonkonferenz Nr. 1, profile: profile_1
2013-02-18 22:47:34.175886 [DEBUG] freeswitch_lua.cpp:370 DBH handle (nil) released.
2013-02-18 22:47:34.175886 [DEBUG] mod_conference.c:8208 override sound-prefix with: /opt/freeswitch/sounds/en/us/callie
2013-02-18 22:47:34.175886 [DEBUG] mod_conference.c:1923 Setup timer success interval: 20  samples: 320
2013-02-18 22:47:34.175886 [DEBUG] mod_conference.c:7121 Raw Codec Activation Success L16@8000hz 1 channel 20ms
```

**Comment by jpawlowski** [comment]
Hm, okay there is a 2nd option "sound-prefix" in section conference to be changed.
This is the default setting for external participants, okay. Can internals keep their language? :D

**Comment by spag** [comment]
fixed in commit dd5566e1a3480d0f000e25470e76e5c0e1d5507f
